pages/api/schedule/ics.tsx

```diff
@@ -0,0 +1,83 @@
+import { firestore } from 'firebase-admin';
+import { NextApiRequest, NextApiResponse } from 'next';
+import initializeApi from '../../../lib/admin/init';
+import { userIsAuthorized } from '../../../lib/authorization/check-authorization';
+import { createEvent } from 'ics';
+initializeApi();
+const db = firestore();
+
+const SCHEDULE_EVENTS = '/schedule-events';
+
+async function getIcalEvent(req: NextApiRequest, res: NextApiResponse) {
+  let event = await db.collection(SCHEDULE_EVENTS).get();
+  console.log(event);
```

Comment:
There are quite a few console logs around; I recommend bumping the ones that are here for debugging down to the [console. debug](https://developer.mozilla.org/en-US/docs/Web/API/console/debug_static) level to make it easier to filter out. Or just remove them from their entity if they are not needed.